### PR TITLE
Feature/object action generator

### PR DESCRIPTION
**Pull request by Lukyio**

### CODE REVIEW

File: client/src/routes/automation/BlocklyComponent.svelte [hunk 1]
```diff
@@ -27,7 +27,8 @@
 </script>
 
 <p>
-    <button on:click={blocklyService.convertToBlock()}>Convert the blocks !</button>
+    <button on:click={blocklyService.convertToBlock()}>TEST: Convert the blocks
```
Comment (asoran) [inline client/src/routes/automation/BlocklyComponent.svelte]:
```suggestion
    <button on:click={blocklyService.convertToBlock}>TEST: Convert the blocks
```
Faut donner une reference de fonction, là ici ta fonction convertToBlock se lance dès que la page se charge  :^)

File: client/src/routes/automation/esidom_generator.ts [hunk 2]
```diff
@@ -231,6 +233,19 @@ interface BlocksGenerator {
         return JSON.stringify(json);
     };
 
+    block.object_action = (blk: Block) => {
+        const dropdown_entities = blk.getFieldValue('Entities');
+        const dropdown_services = blk.getFieldValue('Services');
+
+        const json: BlocklyJSON = {};
+
+        const entity_id = dropdown_entities.split(':')[1];
```
Comment (asoran) [inline client/src/routes/automation/esidom_generator.ts:233]: Care, array[1] can return undefined. Try to check for it and do something ? 🤔

**Review by asoran — CHANGES_REQUESTED** [review]
Please apply the changes

File: client/src/routes/automation/esidom_generator.ts (hunk 2 above)
Comment (Lukyio) [reply client/src/routes/automation/esidom_generator.ts:233]: Check for it then do what ? I don't know.

File: client/src/routes/automation/BlocklyComponent.svelte (hunk 1 above)
Comment (Lukyio) [reply client/src/routes/automation/BlocklyComponent.svelte]: Done

**Review by asoran — APPROVED** [review]
:zap:

File: client/src/routes/automation/esidom_generator.ts (hunk 2 above)
Comment (Nailykk) [reply client/src/routes/automation/esidom_generator.ts:233]: Type is embedded automatically. This should never be undefined unless a code error happend.
Is there a way to send an exception or just ignore this ?
@asoran

**Review by guyts96 — APPROVED** [review]

**Pull request merged**